daml-lf/spec/daml-lf-1.rst

```diff
@@ -675,6 +675,14 @@ Then we can define our kinds, types, and expressions::
        | 'throw' @σ @τ e                            -- ExpThrow: throw exception [Daml-LF ≥ 1.14]
        | 'to_any_exception' @τ e                    -- ExpToAnyException: Turn a concrete exception into an 'AnyException' [Daml-LF ≥ 1.14]
        | 'from_any_exception' @τ e                  -- ExpFromAnyException: Extract a concrete exception from an 'AnyException' [Daml-LF ≥ 1.14]
+       | 'to_interface' @τ₁ @τ₂ e                   -- ExpToInterface: Turn a template value into an interface value [Daml-LF ≥ 1.dev]
```

Comment:
dumb question, does the name on the right hand side (`ExpToInterface` here) only refer to the corresponding inference rule below or should it match something else? I'm asking because in other places we call this `EToInterface` (and even `ExprSumToInterface` in the protobuff Haskell bindings)

Reply:
There are no bad questions!

It's only used here and in the typing rules and operational semantics. We probably should change all of these to `EToInterface` and so on.

Reply:
got it, thank you!